components/notebook-controller/config/overlays/openshift/kustomization.yaml

```
@@ -1,34 +1,43 @@
----
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - ../../base
+- ../../base
 namespace: opendatahub
-commonLabels:
-  app.kubernetes.io/part-of: odh-notebook-controller
-  component.opendatahub.io/name: kf-notebook-controller
-  opendatahub.io/component: "true"
 configurations:
-  - params.yaml
+- params.yaml
 configMapGenerator:
-  - name: kf-notebook-controller-image-parameters
-    env: params.env
-  - name: config
-    behavior: merge
-    literals:
-      - USE_ISTIO=false
-      - ADD_FSGROUP=false
+- envs:
+  - params.env
+  name: kf-notebook-controller-image-parameters
+- behavior: merge
+  literals:
+  - USE_ISTIO=false
+  - ADD_FSGROUP=false
+  name: config
 generatorOptions:
   disableNameSuffixHash: true
-patchesStrategicMerge:
-  - remove_namespace_patch.yaml
-  - manager_openshift_patch.yaml
-  - manager_service_openshift_patch.yaml
-vars:
-- name: odh-kf-notebook-controller-image
-  objref:
+labels:
+- includeSelectors: true
+  pairs:
+    app.kubernetes.io/part-of: odh-notebook-controller
+    component.opendatahub.io/name: kf-notebook-controller
+    opendatahub.io/component: "true"
+patches:
+- path: remove_namespace_patch.yaml
+- path: manager_openshift_patch.yaml
+- path: manager_service_openshift_patch.yaml
+replacements:
+- source:
+    fieldPath: data.odh-kf-notebook-controller-image
     kind: ConfigMap
     name: kf-notebook-controller-image-parameters
-    apiVersion: v1
-  fieldref:
-    fieldpath: data.odh-kf-notebook-controller-image
+    version: v1
+  targets:
+  - fieldPaths:
+    - spec.template.spec.containers.0.image
+    select:
+      group: apps
+      kind: Deployment
+      name: manager
+      namespace: system
```

```suggestion
      name: deployment
```
the name should match : https://github.com/mtchoum1/kubeflow/blob/da8ff0d134b4214ba5cd894692f576d050a7e746/components/notebook-controller/config/manager/manager.yaml#L11
also remove the namespace